.github/workflows/brokenlink-check.yml

```diff
@@ -0,0 +1,15 @@
+name: Links
+
+on:
+  workflow_dispatch:
+
+jobs:
+  my-broken-link-checker:
+    name: Check broken links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
+        uses: ruzickap/action-my-broken-link-checker@v2
+        with:
+          url: https://qiskit.org
+          cmd_params: "--buffer-size=8192 --max-connections=10 --color=always --skip-tls-verification --exclude=\"(stubs|https://qiskit.org/documentation)\" --timeout=60"
```

Comment:
Did you mean `/ecosystem` rather than `/documentation`? It's important that we don't have broken links for `/documentation`. `/ecosystem` is also important, but less so.

Can you please look more into how the `exclude` option works? I'm wondering whether we need to have `https://qiskit.org` in the entry. And I want to ensure it checks all subpages too.

Reply:
Hi @Eric-Arellano, in an offline conversation we thought that this workflow should focus on everything that is not documentation and that the documentation team should replicate this workflow exclusively for documentation since the volume of broken links there is much higher and it would be better to treat it separately. Do you agree with that idea or do you think there is a better approach?

Regarding the base url, it would not be strictly necessary because I think we do not have more links containing the word "documentation" but I wanted to ensure that this is in that particular position with respect to the base url. With this exactly entry, it would check all the page except for documentation and it's subpages.

Reply:
Okay, that's fine with me. But you will still want to exclude `/ecosystem` too. Specifically, any subpages of `/ecosystem`, like `/ecosystem/finance`. `/ecosystem` itself is good to check.